exercise/templates/exercise/_children.html

```diff
@@ -11,18 +11,17 @@
 {% endif %}
 {% elif entry|is_listed %}
 <li>
-  {% if accessible %}
-  {% if entry.is_empty %}
-  {{ entry.name|parse_localization }}
+  {% if accessible and not entry.submittable or exercise_accessible %}
+   {% if entry.is_empty %}
```

Comment:
It looks like you have only changed one if condition here, but because you changed the indentations, many lines show up here in the git diff. Using one space is not good for indentation, and it is good to avoid making such whitespace changes that pollute the diff unless there is a good reason to do it. Either fix the indentations properly or do not change them on existing lines. I think you should remove your indentation changes so that the diff shows only the one changed if condition in the end after squashing.

Reply:
In August there was a problem when my atom changed spaces in tabs or other way around when I saved this file. I tried to fix them manually then. I'll use the proper indentation this time but git diff might still be little dirty.